### PR TITLE
[release-2.2] Organization rename to `stolostron`

### PR DESCRIPTION
**Pull request by dhaiducek**
- https://github.com/stolostron/backlog/issues/18775

### CODE REVIEW

File: deploy/operator.yaml [hunk 1]
```diff
@@ -17,7 +17,7 @@ spec:
         - name: multiclusterhub-operator-pull-secret
       containers:
         - name: governance-policy-template-sync
-          image: quay.io/open-cluster-management/governance-policy-template-sync:latest-dev
+          image: quay.io/stolostron/governance-policy-template-sync:latest-dev
```
Comment (ycao56) [inline deploy/operator.yaml:20]: latest-2.2?

Comment (dhaiducek) [reply deploy/operator.yaml]: Updated. I'll update propagator also--I didn't update the tag there.

File: go.mod [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
+	github.com/stolostron/governance-policy-propagator v0.0.0-20220112203818-b0bc3438ecd7
+	k8s.io/api v0.18.3
+	k8s.io/apimachinery v0.18.3
 	k8s.io/client-go v12.0.0+incompatible
 	k8s.io/klog v1.0.0
-	sigs.k8s.io/controller-runtime v0.5.2
+	sigs.k8s.io/controller-runtime v0.6.0
 )
 
 replace (
```
Comment (ycao56) [inline go.mod:18]: missing replace. old api path still showing up in https://github.com/stolostron/governance-policy-template-sync/blob/b018fe140a06b7e76f83cf6f7cb2ffde00ece322/go.sum#L625

Comment (dhaiducek) [reply go.mod:18]: Shoot--I didn't know it wouldn't replace it here. Thanks!

File: build/run-e2e-tests.sh [hunk 3]
```diff
@@ -16,8 +16,8 @@ if ! which kind > /dev/null; then
     sudo mv ./kind /usr/local/bin/kind
 fi
 echo "Installing ginkgo ..."
-go get github.com/onsi/ginkgo/ginkgo
-go get github.com/onsi/gomega/...
+go get github.com/onsi/ginkgo/ginkgo@v1.14.1
+go get github.com/onsi/gomega/...@v1.10.2
```
Comment (ycao56) [inline build/run-e2e-tests.sh:20]: go get github.com/onsi/ginkgo/ginkgo@1.10.2: github.com/onsi/ginkgo/ginkgo@1.10.2: invalid version: unknown revision 1.10.2

**Comment by dhaiducek** [comment]
This PR is ready for review.

**Review by ycao56 — APPROVED** [review]

**Pull request merged**